rosdep/python.yaml

```diff
@@ -1988,6 +1988,8 @@ python-googleapi:
   fedora: [google-api-python-client]
   gentoo: [dev-python/google-api-python-client]
   ubuntu:
+    bionic: [python-googleapi]
+    focal: [python3-googleapi]
```

Comment:
Following our conventions this key is for Python 2 /any. We should not mix in the explicit python3 version here. It should get a new key `python3-googleapi`

Reply:
Thanks, I'll fix

Reply:
fixed